cockroachcloud/cloud-org-sso.md

```diff
@@ -0,0 +1,126 @@
+
+## Basic SSO
+
+Basic SSO provides flexibility and convenience for your users, and is enabled by default for each {{ site.data.products.db }} organization. With no configuration required, members can sign in using an identity from GitHub, Google, or Microsoft instead of using an email address or password.
+
+Basic SSO has the following differences from [Cloud Organization SSO](#cloud-organization-sso):
+
+  - Customization is not possible.
+  - New authentication methods cannot be added, and existing authentication methods cannot be disabled.
+  - It is not possible to enforce a requirement to use SSO rather than password authentication.
+  - It is not possible to limit the email domains allowed to sign in using a given authentication method.
+  - Autoprovisioning is not supported, and members must be invited before they can sign in.
+  - A member may have only one active authentication method in an organization, but may change it at any time by logging in using a different method. For a member to switch back to using a password, they must be removed and re-invited to your {{ site.data.products.db }} organization. Refer to [Migration of individual members to SSO](#migration-of-individual-members-to-sso).
+
+If your organization needs more flexibility and customization to meet your security and compliance requirements, you can enable and configure [Cloud Organization SSO](#cloud-organization-sso).
+
+## Cloud Organization SSO
+
+Cloud Organization SSO allows you to customize your SSO configuration to meet your organization's security and business requirements:
+
+- Members sign in using a custom URL that allows only the authentication methods that you have configured.
+- Members can sign in using any enabled authentication method, to help reduce the impact of an identity provider outage. If a member signs in using a new method, they are prompted to optionally update their default.
+- Multiple authentication methods can be enabled simultaneously. You can add custom authentication methods that connect to IdPs such as Okta or OneLogin using the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
+- Authentication methods can be disabled, including email authentication. Disabled authentication methods are not available on the custom login URL.
+- [Autoprovisioning](#autoprovisioning) optionally removes the need to invite members to your organization. Autoprovisioning is disabled by default for each SSO authentication method.
+- You can limit the email domains that are allowed to log in, at the level of the individual authentication method. By default, any email domain is allowed.
+
+To enable and configure Cloud Organization SSO, refer to [Configure Cloud Organization SSO](configure-cloud-org-sso.html).
+
+### Restrict SSO authentication by email domain
```

Comment:
I'm not sure what you mean about the point not being clear. If you are example.com, then you might want to restrict SSO to Google to @example.com so that Gmail users can't sign in or be invited.

Reply:
Now I see what you mean. I've removed a lot of this detail from this topic as it is already in the task.